resources/views/profile/edit.blade.php

```diff
@@ -4,14 +4,96 @@
     $title = __('navbar.account');
 @endphp
 
+@push('scripts')
+    <script src="{{ @asset('js/auth/profile.js') }}" defer></script>
+@endpush
+
 @section('content')
-    <div class="w-full max-w-md mx-auto p-6 items-center" style="inline-size: fit-content;">
-        <form action="{{ route('logoutpost') }}" method="POST">
-            @csrf
-            <button type="submit"
-                    class="py-3 px-4 inline-flex items-center gap-x-2 text-sm font-semibold rounded-lg border border-transparent bg-gray-100 text-gray-500 hover:bg-gray-200 disabled:opacity-50 disabled:pointer-events-none dark:bg-white/10 dark:hover:bg-white/20 dark:text-gray-400 dark:hover:text-gray-300 dark:focus:outline-none dark:focus:ring-1 dark:focus:ring-gray-600">
-                {{__('auth/auth.logout')}} {{ Auth::user()->getEmail() }}
-            </button>
-        </form>
+    <div class="grid gap-4">
+        <div class="flex flex-row justify-between items-center">
+            <h1 class="text-4xl dark:text-white">Welkom!</h1>
+            <div>
+                <form action="{{ route('logoutpost') }}" method="POST">
+                    @csrf
+                    <button type="submit"
```

Comment:
De button ziet er momenteel zo uit in darkmode:

![image](https://github.com/Avans-290IN4SOj-Scouting/Scouting/assets/58737124/e657b02e-8c3f-49b2-a73c-2f9092ad80a8)

en zo als ik eroverheen hover:

![image](https://github.com/Avans-290IN4SOj-Scouting/Scouting/assets/58737124/04ae251a-b88f-4c74-bf09-6fd5d0c0d767)

Persoonlijk zou ik de kleuren van de non-hover donkerder maken, zodat het meer in toon is met de hover variant. Hier is een voorbeeld:

![image](https://github.com/Avans-290IN4SOj-Scouting/Scouting/assets/58737124/5d4be5bf-f322-40fa-911f-0c0606e1f18f)

```html
dark:bg-red-600 dark:text-red-100